src/services/rhsm/__tests__/__snapshots__/rhsmConstants.test.js.snap

```diff
@@ -47,18 +47,31 @@ exports[`RHSM Constants should have specific properties: all exported constants
     "QUARTERLY": "Quarterly",
     "WEEKLY": "Weekly",
   },
+  "RHSM_API_QUERY_INVENTORY_HOSTS_SORT_TYPES": {
+    "CORES": "cores",
+    "CORE_HOURS": "core_hours",
+    "HARDWARE": "hardware_type",
+    "INSTANCE_HOURS": "instance_hours",
+    "LAST_SEEN": "last_seen",
+    "MEASUREMENT": "measurement_type",
+    "NAME": "display_name",
+    "SOCKETS": "sockets",
+  },
   "RHSM_API_QUERY_INVENTORY_SORT_DIRECTION_TYPES": {
     "ASCENDING": "asc",
     "DESCENDING": "desc",
   },
   "RHSM_API_QUERY_INVENTORY_SORT_TYPES": {
     "BILLING_PROVIDER": "billing_provider",
-    "CORES": "Cores",
+    "CORES": "cores",
+    "CORE_HOURS": "core_hours",
     "CORE_SECONDS": "Core-seconds",
-    "INSTANCE_HOURS": "Instance-hours",
+    "HARDWARE": "hardware_type",
+    "INSTANCE_HOURS": "instance_hours",
     "LAST_SEEN": "last_seen",
+    "MEASUREMENT": "measurement_type",
     "NAME": "display_name",
-    "SOCKETS": "Sockets",
+    "SOCKETS": "sockets",
```

Comment:
required:
confirm this isn't going to be an issue with the sort-types check... make sure it's case insensitive

Reply:
confirmed, it does appear this causes issues with `Cores` vs `cores` and `Sockets` vs `sockets`.